hlc_wp_saw/config.cpp

```diff
@@ -441,197 +486,211 @@ class CfgWeapons {
-{
-    author = "Toadie";
-    model = "\hlc_wp_saw\mesh\minimi_para\minimi_railed.p3d";
-    displayName = "FN Minimi Para (Short/RIS)";
-    inertia = 0.69;
-    __DEXTERITY(6.94, 0);
-    class WeaponSlotsInfo : WeaponSlotsInfo {
-        mass = 145;
-        class CowsSlot : asdg_OpticRail1913_short_MG {};
+    {
+        author = "Toadie";
+        model = "\hlc_wp_saw\mesh\minimi_para\minimi_railed.p3d";
+        displayName = "FN Minimi Para         [13.7 inch:RIS] ";
+        inertia = 0.69;
+        __DEXTERITY(6.94, 0);
+        class WeaponSlotsInfo : WeaponSlotsInfo 
+        {
+            mass = 146.9;
+            class CowsSlot : asdg_OpticRail1913_short_MG {};
+        };
     };
-};
+
     class hlc_lmg_minimipara_long : hlc_lmg_minimipara
-{
-    author = "Toadie";
-    model = "\hlc_wp_saw\mesh\minimi_para\minimi_para_longer.p3d";
-    displayName = "FN Minimi Para (Long)";
-    initspeed = -1;
-    inertia = 0.68;
-    __DEXTERITY(6.85, 0);
-    class WeaponSlotsInfo : WeaponSlotsInfo {
-        mass = 137;
+    {
+        author = "Toadie";
+        model = "\hlc_wp_saw\mesh\minimi_para\minimi_para_longer.p3d";
+        displayName = "FN Minimi Para         [18.3 inch]";
+        initspeed = -1;
+        AB_barrelTwist = 7;
+        AB_barrelLength = 18.3;
+        agm_bipod = 1;
+        ACE_barrelTwist = 178;
+        ACE_barrelLength = 465;
+        inertia = 0.68;
+        __DEXTERITY(6.85, 0);
+        class WeaponSlotsInfo : WeaponSlotsInfo 
+        {
+            mass = 150.9;
+        };
     };
-};
-    class hlc_lmg_minimipara_long_railed : hlc_lmg_minimipara
-{
-    author = "Toadie";
-    model = "\hlc_wp_saw\mesh\minimi_para\minimi_para_longer_railed.p3d";
-    displayName = "FN Minimi Para (Long/RIS)";
-    initspeed = -1;
-    inertia = 0.69;
-    __DEXTERITY(6.94, 0);
-    class WeaponSlotsInfo : WeaponSlotsInfo {
-        mass = 145;
-        class CowsSlot : asdg_OpticRail1913_short_MG {};
+    class hlc_lmg_minimipara_long_railed : hlc_lmg_minimipara_long
+    {
+        author = "Toadie";
+        model = "\hlc_wp_saw\mesh\minimi_para\minimi_para_longer_railed.p3d";
+        displayName = "FN Minimi Para         [18.3 inch:RIS] ";                                // this variant should have a shorter barrel ( maybe 16 inch ) and lighter buttstock. the model has the same barrel ( 18.3 inch ) though. weight is adjusted to appereance.
+        initspeed = -1;
+        inertia = 0.69;
+        __DEXTERITY(6.94, 0);
+        class WeaponSlotsInfo : WeaponSlotsInfo 
+        {
+            mass = 155.6;
+            class CowsSlot : asdg_OpticRail1913_short_MG {};
+        };
     };
-};
-    class hlc_lmg_minimi : hlc_lmg_minimipara {
+    class hlc_lmg_minimi : hlc_lmg_minimipara_long
+    {
         author = "Toadie";
         model = "\hlc_wp_saw\mesh\minimi_para\minimi_longer.p3d";
-        displayName = "FN Minimi (Long)";
+        displayName = "FN Minimi             [18.3 inch]";

```

Comment:
As a related side-note- Why the square brackets?

Reply:
so it picks directly into the eye and is easier to be distincted.
Doesnt have to be those bracked, but the curvy brackets are used for additional information, not, as I see it: "important information".